etc/config-template.toml

```diff
@@ -162,14 +162,22 @@
 
 # Interval to check whether start manual compaction for a region,
 # region-compact-check-interval = "5m"
+
 # Number of regions for each time to check.
 # region-compact-check-step = 100
+
 # The minimum number of delete tombstones to trigger manual compaction.
 # region-compact-min-tombstones = 10000
+
+# The minimum percentage of delete tombstones to trigger manual compaction.
+# Should between 1 and 100.
+# region-compact-tombstones-ratio = 30
```

Comment:
I think it should be region-compact-tombstones-percent if we use 1~100.

Reply:
We can change to a f64 and use "0.3".

Reply:
We also need to document the precondition of triggering the compaction, i.e. `numer || ratio` or `number && ratio`.